phpBB/includes/functions.php

```diff
@@ -4969,6 +4969,8 @@ function page_header($page_title = '', $display_online_list = false, $item_id =
 		}
 	}
 
+	$user->update_session_infos();
```

Comment:
`$user->update_session_page` will be checked in `update_session_infos();`
Also there is many other variables you could use to prevent that from happening but whatever. If it's in page_header you'll have to do the same anyway - it really is not that much of a difference.